### ISSUE TITLE
Project active?

### ISSUE THREAD

**Issue by peterfortuin** [cloudendpoints/endpoints-java#202]
I really like this library and use in one of my projects. But I started to wonder if this is still active maintained? Or is it better to migrate to something else?

**Comment by peterfortuin** [comment]
Since this issue is not responded to I think the project is dead. Time to migrate to something else. Somebody has suggestions to something similar?

**Comment by deepfriedbrain** [comment]
Have you found any other options @peterfortuin ?

**Comment by peterfortuin** [comment]
Not really. I'm experimenting with Kotlin with Ktor. Going to figure out if OpenApi/Swagger﻿ is the way forward.

**Comment by deepfriedbrain** [comment]
Hope you will share the experience after you've come to some conclusion.
